### ISSUE TITLE
Manjaro (Arch) ckan won't start with Exec format error

### ISSUE THREAD

**Issue by Jorval** [KSP-CKAN/CKAN#2088]
Hey,
I installed ckan form AUR. First the normal Version, hitting this error. then deinst and tried the git version from AUR same result.

**CKAN Version:** 
aur/ckan 1.22.3-1
aur/ckan-git 1.22.3.r0.g64309722-1

**Operating System:** 
Manjaro

**The issue you are experiencing:** 
starting ckan didn't work. Getting an Error the binary has an Error in execfile (Sorry error is dropped in German!)
bash: /usr/bin/ckan: Kann die Binärdatei nicht ausführen: Fehler im Format der Programmdatei

**How to recreate this issue:** 
trying to start ckan ;)

**CKAN error codes (if applicable):** 
did a strace. such iam not a c developer it's ... hope you know what to do
strace:
strace ckan
execve("/usr/bin/ckan", ["ckan"], 0x7ffcf9430810 /* 72 vars */) = -1 ENOEXEC (Exec format error)
fstat(2, {st_mode=S_IFCHR|0620, st_rdev=makedev(136, 0), ...}) = 0
write(2, "strace: exec: Exec format error\n", 32strace: exec: Exec format error
) = 32
getpid()                                = 7774
exit_group(1)                           = ?
+++ exited with 1 +++

**Comment by StollD** [comment]
I am not sure, but to me it looks like the AUR package just copies ckan into /usr/bin/ and makes it executable. Because CKAN is a .NET application, it cannot be run directly, you have to run it through mono. The package however seems to treat it like a native executable file.

TL;DR: Could you try to run `mono /usr/bin/ckan` and see if that works?

**Comment by Jorval** [comment]
OK, feeling a bit dumb now!
yes, that worked. made myself a .desktop and now it starts smooth (except that ugly mono look).
thanks and sorry bothering you.
